Applications/WebEditor.qmd

```diff
@@ -63,48 +63,96 @@ Based on their applications user can build their model by simple drag and drop m
+    a. To start a task within the openEO web editor, your first step involves ensuring you have the necessary dataset for the analysis. Thus, you must ensure the required dataset exists within the openEO database. You can do this by exploring/searching the list of available datasets from the sidebar of the interface.
+    
+    b. In our example, we want to calculate the Normalized Difference Vegetation Index (NDVI), so the Sentinel 2 L2A collection will be used. Once you find this collection, the next step is to drag and drop it into the Process Editor for further operations.
+        
+    ![](_images/webeditor/drag_collection.png)
+
+
+    Now, once you load collection, there are several parameters you need to specify, including the area of interest, the temporal extent, and the selection of bands. When you click on the collection, a window pops up, where you can define the necessary parameters for subsequent processing. 
+    
+    c. To specify the area of interest, you can choose between generating a bounding box or importing your spatial extent by dragging and dropping GeoJSON or KML files onto the map view. Here, we've provided an example involving a bounding box. There is also an option to "No filter", that will include all the data in the datacube.
+
+    ![](_images/webeditor/define_bbox.png)
 
+    d. Another parameter is Temporal extent, that imits the data to load from the collection to the specified  temporal interval.
 
-![](_images/gif/load_collection.gif)
+    ![](_images/webeditor/define_temporal.png)
+
+    e. In our case, for NDVI calculation, we have specifically chosen the Red band (B04) and the Near-Infrared (NIR) band (B08). 
+        
+    ![](_images/webeditor/define_bands.png)
 
 2. Apply Processes
     
-    Eventually, the next step involves implementing essential processes, ranging from straightforward operations like adding bands to more complex tasks such as importing or defining user-defined functions (UDFs). In the following clip, a reduce_dimension() process is employed to eliminate the temporal dimension by selecting the maximum value. The same process is utilised to reduce the band dimension after executing a series of addition, subtraction, and division operations necessary for the NDVI calculation.
+    Eventually, the next step involves implementing essential processes, ranging from straightforward operations like adding bands to more complex tasks such as importing or defining user-defined functions (UDFs). 
+
+    In this task, we're using a specific process called `reduce_dimension` as a key part of the operation. This process helps us simplify our collection: first, we use it to reduce temporal dimension and second, to deal with the various bands.
+            
+    ![](_images/webeditor/reduce_dimension.png)
+
+    In the initial process, the `reduce_dimension` algorithm eliminates the temporal dimension by selecting the maximum value using the max function.
```

Comment:
Note that, scientifically speaking, it is pretty weird to first do a temporal reduction of your raw sensor values and do NDVI calculation afterwards because you will be calculating the NDVI from red and nir values that might be unrelated. I think you first should do NDVI calculation and then taking the temporal max.